service/src/main/java/uk/gov/hmcts/reform/fpl/model/DirectionResponse.java

```diff
@@ -11,11 +11,13 @@
 import java.util.List;
 import java.util.UUID;
 
+//TODO: rename variable assignee. This is the Role of the person complying.
```

Comment:
Can we rename it sooner rather than later? If we don't rename it now (or separate PR), we will need data transformation which is a pain.

Reply:
for a separate PR